src/components/UI/Empty.tsx

```diff
@@ -42,7 +42,7 @@ const generateDescriptionNode = (rawDescription: string | undefined, size: SIZE)
   </span>
 );
 
-const Empty: FunctionComponent<OwnProps> = ({
+const Empty: FC<OwnProps> = ({
```

Comment:
the FC pattern is deprecated. 
https://github.com/facebook/create-react-app/pull/8177